routes/FeedbackListController.php

```diff
@@ -25,6 +25,7 @@ public static function get(): :xhp {
       # Append the applicant to the table as a new row
       $table_body->appendChild(
         <tr class={$feedback->getComments() != '' ? "success" : ""}>
```

Comment:
And note that this means you will have to add the id element (to the id table entry) on the reviewerListController as well.